### ISSUE TITLE
cannot import name 'VoxelGeneratorV2'

### ISSUE THREAD

**Issue by zhongzhuonan** [traveller59/second.pytorch#346]
```
  File "/home/zzn/Object_Detection/second.pytorch/second/builder/voxel_builder.py", line 3, in <module>
    from spconv.utils import VoxelGeneratorV2
ImportError: cannot import name 'VoxelGeneratorV2'

```
Anaconda
apex                      0.1                      pypi_0    pypi
blas                      1.0                         mkl  
ca-certificates           2020.1.1                      0  
certifi                   2019.11.28               py36_0  
cffi                      1.14.0           py36h2e261b9_0  
cloudpickle               1.3.0                      py_0  
cudatoolkit               10.0.130                      0  
cycler                    0.10.0                   py36_0  
cytoolz                   0.10.1           py36h7b6447c_0  
dask-core                 2.11.0                     py_0  
dbus                      1.13.12              h746ee38_0  
decorator                 4.4.1                      py_0  
expat                     2.2.6                he6710b0_0  
fire                      0.2.1                    pypi_0    pypi
fontconfig                2.13.0               h9420a91_0  
freetype                  2.9.1                h8a8886c_1  
glib                      2.63.1               h5a9c865_0  
gst-plugins-base          1.14.0               hbbd80ab_1  
gstreamer                 1.14.0               hb453b48_1  
icu                       58.2                 h9c2bf20_1  
imageio                   2.6.1                    py36_0  
intel-openmp              2020.0                      166  
jpeg                      9b                   h024ee3a_2  
kiwisolver                1.1.0            py36he6710b0_0  
ld_impl_linux-64          2.33.1               h53a641e_7  
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1                hd88cf55_4  
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_0  
libpng                    1.6.37               hbc83047_0  
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.1.0                h2733197_0  
libuuid                   1.0.3                h1bed415_2  
libxcb                    1.13                 h1bed415_1  
libxml2                   2.9.9                hea5a465_1  
llvmlite                  0.31.0           py36hd408876_0  
matplotlib                3.1.3                    py36_0  
matplotlib-base           3.1.3            py36hef1b27d_0  
mkl                       2020.0                      166  
mkl-service               2.3.0            py36he904b0f_0  
mkl_fft                   1.0.15           py36ha843d7b_0  
mkl_random                1.1.0            py36hd6b4f25_0  
ncurses                   6.2                  he6710b0_0  
networkx                  2.4                        py_0  
ninja                     1.9.0            py36hfd86e86_0  
numba                     0.48.0           py36h0573a6f_0  
numpy                     1.18.1           py36h4f9e942_0  
numpy-base                1.18.1           py36hde5b4d6_1  
olefile                   0.46                     py36_0  
opencv-python             4.2.0.32                 pypi_0    pypi
openssl                   1.1.1d               h7b6447c_4  
pcre                      8.43                 he6710b0_0  
pillow                    7.0.0            py36hb39fc2d_0  
pip                       20.0.2                   py36_1  
protobuf                  3.11.3                   pypi_0    pypi
pycparser                 2.19                     py36_0  
pyparsing                 2.4.6                      py_0  
pyqt                      5.9.2            py36h05f1152_2  
python                    3.6.10               h0371630_0  
python-dateutil           2.8.1                      py_0  
pytorch                   1.2.0           py3.6_cuda10.0.130_cudnn7.6.2_0    pytorch
pywavelets                1.1.1            py36h7b6447c_0  
qt                        5.9.7                h5867ecd_1  
readline                  7.0                  h7b6447c_5  
scikit-image              0.16.2           py36h0573a6f_0  
scipy                     1.4.1            py36h0b6359f_0  
setuptools                45.2.0                   py36_0  
sip                       4.19.8           py36hf484d3e_0  
six                       1.14.0                   py36_0  
spconv                    1.0                      pypi_0    pypi
sqlite                    3.31.1               h7b6447c_0  
tensorboardx              2.0                      pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
tk                        8.6.8                hbc83047_0  
toolz                     0.10.0                     py_0  
torchvision               0.4.0                py36_cu100    pytorch
tornado                   6.0.3            py36h7b6447c_3  
wheel                     0.34.2                   py36_0  
xz                        5.2.4                h14c3975_4  
zlib                      1.2.11               h7b6447c_3  
zstd                      1.3.7                h0b5b093_0 

**I successfully installed spconv
If you know anything, please let me know. Thank you.**

**Comment by JingyaLiu** [comment]
I encounter the same issue. Did you solve it?

**Comment by zhongzhuonan** [comment]
> I encounter the same issue. Did you solve it?

sorry,I didn't solve it .

**Comment by chasingw** [comment]
use spconv a6ae896 worked

**Comment by zhongzhuonan** [comment]
> use spconv a6ae896 worked

thank you

**Comment by Zone-w** [comment]
I encounter the same issue. Did you solve it?

**Comment by sixcomicpen** [comment]
> spconv a6ae896
请问怎么使用spconv a6ae896

**Comment by goodChiken** [comment]
> use spconv a6ae896 worked

Sorry to ditribute u, but how to use the spconv a6ae896.

**Comment by chasingw** [comment]
> > use spconv a6ae896 worked
> 
> Sorry to ditribute u, but how to use the spconv a6ae896.

just clone spconv repo and `git check out a6ae896`

**Comment by rohithsaro** [comment]
> 

Not sure what you mean by this.

**Comment by rohithsaro** [comment]
Sorry, i did not get how to solve this issue. 
Could you please be more clear ? 

Thank you.

**Comment by yy-at-here** [comment]
@traveller59 
I also encountered the same issue.
I checked the branch whose commit id=a6ae896, then it was found to be the branch of spconv.1.1.
How can I use  VoxelGeneratorV2 in spcon.2.x ?
Now I'm using PyPI version of spconv. Should I do `git clone ~~/spconv` and add it to pip manually ?

Thank you.

**Comment by lixinghe1999** [comment]
I occasionally find it is enough to just copy&paste the older "VoxelGenerator" into the newer code, since VoxelGenerator is just an independent class.

**Comment by yuezhao1** [comment]
> I occasionally find it is enough to just copy&paste the older "VoxelGenerator" into the newer code, since VoxelGenerator is just an independent class.

You are right. I successfully pasted the previous versions of the "VoxelGenerator" and "VoxelGeneratorV2" classes into the new version. thank you

**Comment by yuezhao1** [comment]
> I occasionally find it is enough to just copy&paste the older "VoxelGenerator" into the newer code, since VoxelGenerator is just an independent class.

You are right. I successfully pasted the previous versions of the "VoxelGenerator" and "VoxelGeneratorV2" classes into the new version. thank you

**Comment by Anna1ximi** [comment]
> > I occasionally find it is enough to just copy&paste the older "VoxelGenerator" into the newer code, since VoxelGenerator is just an independent class.
> 
> You are right. I successfully pasted the previous versions of the "VoxelGenerator" and "VoxelGeneratorV2" classes into the new version. thank you

@yuezhao1 hey! I want to know how this step is done, I don't quite understand "pasted the previous versions of the "VoxelGenerator" and "VoxelGeneratorV2" classes into the new version" ,can you give me more details? Thanks!!!

**Comment by sidiangongyuan** [comment]
> > I occasionally find it is enough to just copy&paste the older "VoxelGenerator" into the newer code, since VoxelGenerator is just an independent class.
> 
> You are right. I successfully pasted the previous versions of the "VoxelGenerator" and "VoxelGeneratorV2" classes into the new version. thank you

Can u share the code ? And could please tell me  how to do with pasting?   I have the same question

**Comment by leo038** [comment]
> I occasionally find it is enough to just copy&paste the older "VoxelGenerator" into the newer code, since VoxelGenerator is just an independent class.

resolved, thank you!
